### PR TITLE
LPS-125884

### PR DESCRIPTION
**Pull request by brunoquei2105**

### CODE REVIEW

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@brunoquei2105 SF will not allow the use of `*` in import statements. All classes must be imported with their full path.

**Comment by brunoquei2105** [comment]
ci:test:sf

**Comment by brunoquei2105** [comment]
ci:test:relevant

**Comment by brunoquei2105** [comment]
ci:test:sf

**Comment by brunoquei2105** [comment]
ci:test:relevant

**Comment by brunoquei2105** [comment]
@drewbrokke

**Comment by brunoquei2105** [comment]
ci:test:sf

**Comment by brunoquei2105** [comment]
ci:test:relevant

File: modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/portlet/RolesAdminPortlet.java [hunk 1]
```diff
@@ -231,29 +232,37 @@ public Role editRole(
...
+				actionResponse.sendRedirect(redirect);
 
-			return role;
+				return role;
+			}
+			catch (Exception exception) {
+				if (exception instanceof DataLimitExceededException) {
+					SessionErrors.add(actionRequest, exception.getClass());
+				}
+			}
```
Comment (drewbrokke) [inline modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/portlet/RolesAdminPortlet.java:265]: @brunoquei2105 what happens to the `Exception` if it is a different type from `DataLimitExceededException`? To me it looks like it gets "swallowed", meaning it is neither handled, logged, nor re-thrown, meaning we'd have no idea what went wrong.

My suggestion would be to re-throw the exception to resume the original behavior.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@brunoquei2105 this is looking pretty good, but are you planning to include the change for Users as well? Is it because the User data limit exception is not thrown?

Also, just one comment inline.

**Pull request closed without merging**